src/bundle/Resources/public/scss/_anchor-navigation.scss

```
@@ -0,0 +1,74 @@
+.ibexa-anchor-navigation-menu {
+    margin: 0;
+    padding: calculateRem(16px);
+    list-style: none;
+
+    &__btn {
+        display: inline-flex;
+        align-items: center;
+        width: 100%;
+        padding: calculateRem(11px) calculateRem(15px);
+        border: calculateRem(1px) solid transparent;
+        border-radius: $ibexa-border-radius;
+
+        &:before {
```

CSS3 syntax - here and in other places
```suggestion
        &::before {
```
ref. https://developer.mozilla.org/en-US/docs/Web/CSS/::before#syntax